input/pagecontent/conformance.md

```diff
@@ -1,18 +1,8 @@
-An [AU eRequesting Placer](ActorDefinition-au-erequesting-actor-placer.html):
+Each AU eRequesting actor as defined in [Actors and Capabilities](capability-statements.html):
 - **SHALL** implement the [mandatory](#mandatory-elements) and/or [Must Support](#must-support-and-obligation) requirements for the elements defined as such in the implemented AU eRequesting profiles.
```

Comment:
This slightly differs from the FHIR-46806 resolution, should say "... elements in the AU eRequesting profiles"

Reply:
Amended, thanks.